### PR TITLE
chore: remove counters library from the contract code.

### PR DESCRIPTION
**Pull request by EmperorOrokuSaki**
Discussion: #129

### CODE REVIEW

File: contracts/contracts/FleekERC721.sol [hunk 1]
```diff
@@ -133,8 +133,8 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl, Fl
 
         // The mint interaction is considered to be the first build of the site. Updates from now on all increment the currentBuild by one and update the mapping.
         app.currentBuild = 0;
-        app.builds[0] = Build(commitHash, gitRepository);
-
+        app.builds[0] = Build(commitHash, gitRepository);    
+
```
Comment (jsonsivar) [inline contracts/contracts/FleekERC721.sol:137]: Just some whitespace to clear

**Review by jsonsivar — APPROVED** [review]
LGTM

File: contracts/contracts/FleekERC721.sol [hunk 2]
```diff
@@ -86,6 +84,7 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl, Fl
     function initialize(string memory _name, string memory _symbol) public initializer {
         __ERC721_init(_name, _symbol);
         __FleekAccessControl_init();
+        _appIds = 0;
```
Comment (zoruka) [inline contracts/contracts/FleekERC721.sol:87]: Do we need to initialize also in the AccessControl side?

**Review by zoruka — APPROVED** [review]

File: contracts/contracts/FleekERC721.sol (hunk 2 above)
Comment (EmperorOrokuSaki) [reply contracts/contracts/FleekERC721.sol:127]: In mappings, the default value is always zero.

**Pull request merged**